lib/diagnostic.rb

```diff
@@ -43,7 +50,11 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+Class Developer < Person
+  def solve_problems
+    put 'think think think'
```

Comment:
2 things: 1) in Ruby, it's `puts` not `put`; 2) we want the string to be _returned_, which, if you'll remember, if fundamentally different from printing to the console